easybuild/easyconfigs/l/LLVM/LLVM-5.0.0-intel_constexpr.patch

```diff
@@ -0,0 +1,22 @@
+--- lib/IR/Attributes.cpp.orig	2018-02-12 10:42:20.493854152 +0000
```

Comment:
@edmondac Please include short description + reference to upstream source for this patch, or mention author.